CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
   - Fixed PowerMethod for square/non-square, complex/float matrices with stopping criterion.
   - CofR image_sharpness improved for large datasets
   - Geometry alignmentment fix for 2D datasets
-  - CGLS update for sapyb to enable complex data
+  - CGLS update for sapyb to enable complex data, bugfix in use of initial 
```

Comment:
Is this relevant?

Reply:
I suppose yes, as it's the PR and tells that initial wasn't used correctly before?